### PR TITLE
Upload chinese translate

### PR DESCRIPTION
**Pull request by SummonHIM**

### CODE REVIEW

**Comment by Bartuzen** [comment]
Thanks a lot for your contribution 😊 I will do some checks and merge it very soon.

File: app/src/main/res/values-zh-rCN/strings.xml [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+    <string name="torrent_overview_last_seen_complete">最后完整可见</string>
+    <string name="torrent_overview_connections">连接</string>
+    <string name="torrent_overview_connections_format">%1$d （最大%2$d）</string>
+    <string name="torrent_overview_seeds">种子</string>
+    <string name="torrent_overview_seeds_format">%1$d （总计 %2$d）</string>
+    <string name="torrent_overview_peers">用户</string>
+    <string name="torrent_overview_peers_format">@string/torrent_overview_seeds_format</string>
+    <string name="torrent_overview_wasted">已丢弃</string>
+
+    <string name="torrent_files_directory_format"><![CDATA[<b>Directory: </b>]]> %1$s</string>
```
Comment (Bartuzen) [inline app/src/main/res/values-zh-rCN/strings.xml:301]: I think you forgot to translate `Directory` in this one.

Comment (SummonHIM) [reply app/src/main/res/values-zh-rCN/strings.xml:301]: Oh, i think it's rss feed code(xml) at first.

Comment (SummonHIM) [reply app/src/main/res/values-zh-rCN/strings.xml]: updated

**Pull request merged**